### PR TITLE
Update hosted_sizing_guidance.adoc

### PR DESCRIPTION
**Pull request by dockerymick**
Testing unicode symbol

### CODE REVIEW

File: clusters/hosted_control_planes/hosted_sizing_guidance.adoc [hunk 1]
```diff
@@ -32,8 +32,8 @@ Request-based sizing provides a maximum number of hosted control planes that can
 
 For example, to demonstrate resource scaling with API load, consider the following workload configuration:
 
-* One hosted cluster with 9 workers (8 vCPU, 32 GiB each), using the KubeVirt provider
-* Benchmark tool: link:https://github.com/cloud-bulldozer/kube-burner[Kube-burner]
+ One hosted cluster with 9 workers (8 vCPU, 32 GiB each), using the KubeVirt provider
+ Benchmark tool: link:https://github.com/cloud-bulldozer/kube-burner[Kube-burner]
```
Comment (dockerymick) [inline clusters/hosted_control_planes/hosted_sizing_guidance.adoc]: Accidentally removed when i did a Find in the file
```suggestion
 * One hosted cluster with 9 workers (8 vCPU, 32 GiB each), using the KubeVirt provider
 * Benchmark tool: link:https://github.com/cloud-bulldozer/kube-burner[Kube-burner]
```

File: clusters/hosted_control_planes/hosted_sizing_guidance.adoc [hunk 2]
```diff
@@ -32,8 +32,8 @@ Request-based sizing provides a maximum number of hosted control planes that can
 
 For example, to demonstrate resource scaling with API load, consider the following workload configuration:
 
-* One hosted cluster with 9 workers (8 vCPU, 32 GiB each), using the KubeVirt provider
-* Benchmark tool: link:https://github.com/cloud-bulldozer/kube-burner[Kube-burner]
+ * One hosted cluster with 9 workers (8 vCPU, 32 GiB each), using the KubeVirt provider
```
Comment (dockerymick) [inline clusters/hosted_control_planes/hosted_sizing_guidance.adoc]:
```suggestion
* One hosted cluster with 9 workers (8 vCPU, 32 GiB each), using the KubeVirt provider
```

File: clusters/hosted_control_planes/hosted_sizing_guidance.adoc [hunk 3]
```diff
@@ -33,7 +33,7 @@ Request-based sizing provides a maximum number of hosted control planes that can
 For example, to demonstrate resource scaling with API load, consider the following workload configuration:
 
 * One hosted cluster with 9 workers (8 vCPU, 32 GiB each), using the KubeVirt provider
-* Benchmark tool: link:https://github.com/cloud-bulldozer/kube-burner[Kube-burner]
+ * Benchmark tool: link:https://github.com/cloud-bulldozer/kube-burner[Kube-burner]
```
Comment (dockerymick) [inline clusters/hosted_control_planes/hosted_sizing_guidance.adoc]:
```suggestion
* Benchmark tool: link:https://github.com/cloud-bulldozer/kube-burner[Kube-burner]
```

**Review by lahinson — APPROVED** [review]
lgtm

**Pull request merged**